src/dmd/mars.d

```diff
@@ -2062,6 +2062,23 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             params.warnings = DiagnosticReporting.inform;
         else if (arg == "-O")   // https://dlang.org/dmd.html#switch-O
             params.optimize = true;
+        else if (startsWith(p + 1, "message-style="))
+        {
+            const style = p + 1 + "message-style=".length;
+
+            if (strncmp(style, "msvc", 4) == 0)
+            {
+                params.messageStyle = MessageStyle.msvc;
```

Comment:
Maybe shouldn't be called `msvc` unless we make an effort to be consistent with MSVC.

Reply:
fair enough. "default"?

Reply:
How about `dmd`, so that it's meaningful even for other compilers using the DMD frontend?

Reply:
seems a bit circular

Reply:
I was thinking that if the flag were to end up in `gdc`, where the default is the GNU style, then `=default` to enable the DMD style would be rather confusing.

Reply:
Since it looks like we're bikeshedding, maybe `=digitalmars`? :)

Reply:
done